woocommerce/woocommerce
# CYS - reset products and pattern when the site doesn't have AI generated content

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





When the AI is offline, the store should use default products and pattern only when the store doesn't have AI generated content.

Closes #42906 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure you have WooCommerce Beta Tester installed.
2. Enable the customize-store feature flag in Tools -> WCA Test Helper -> Features.
3. Go to WooCommerce -> Home and click on Customize your store task.
4. Set up the store.
5. Ensure that the products and images are generated
6. Go to WooCommerce -> Home and click on Customize your store task.
7. Open your browser's inspector and use Chrome's content override [feature](https://developer.chrome.com/docs/devtools/overrides/) to change the content of https://status.openai.com/api/v2/status.json call. Change status.indicator to major.
8. Refresh the page
9. Setup the store
10. Ensure that the products and images are the same as in step 5.




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

CYS - Reset products and pattern when the site doesn't have AI generated content.

#### Comment 

</details>
